src/main/java/net/minecraftforge/client/event/ModelEvent.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.client.event;
+
+import com.google.common.base.Preconditions;
+import net.minecraft.client.resources.model.BakedModel;
+import net.minecraft.client.resources.model.ModelBakery;
+import net.minecraft.client.resources.model.ModelManager;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraftforge.client.model.geometry.IGeometryLoader;
+import net.minecraftforge.eventbus.api.Cancelable;
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraftforge.fml.LogicalSide;
+import net.minecraftforge.fml.ModLoadingContext;
+import net.minecraftforge.fml.event.IModBusEvent;
+import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
+import org.jetbrains.annotations.ApiStatus;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Houses events related to models.
+ */
+public abstract class ModelEvent extends Event
+{
+    @ApiStatus.Internal
+    protected ModelEvent()
+    {
+    }
+
+    /**
+     * Fired when the {@link ModelManager} is notified of the resource manager reloading.
+     * Called after model registry is setup, but before it's passed to BlockModelShapes.
+     *
+     * <p>This event is not {@linkplain Cancelable cancellable}, and does not {@linkplain HasResult have a result}. </p>
+     *
+     * <p>This event is fired on the {@linkplain FMLJavaModLoadingContext#getModEventBus() mod-specific event bus},
+     * only on the {@linkplain LogicalSide#CLIENT logical client}. </p>
+     */
+    public static class BakingCompleted extends ModelEvent implements IModBusEvent
+    {
+        private final ModelManager modelManager;
+        private final Map<ResourceLocation, BakedModel> models;
+        private final ModelBakery modelBakery;
+
+        @ApiStatus.Internal
+        public BakingCompleted(ModelManager modelManager, Map<ResourceLocation, BakedModel> models, ModelBakery modelBakery)
+        {
+            this.modelManager = modelManager;
+            this.models = models;
+            this.modelBakery = modelBakery;
+        }
+
+        /**
+         * @return the model manager
```

Comment:
This should use the inline `{@return}` tag.
```suggestion
         * {@return the model manager}
```